common/diplomatic_actions/babylon_diplomatic_actions.txt

```diff
@@ -36,6 +34,97 @@
 	}
 }
 
+player_diplomatic_backlash = {
+	requires_approval = yes
+	should_notify_third_parties = yes
+	show_effect_in_tooltip = yes
+	show_confirmation_box = no
+	is_hostile = yes
+
+	potential = {
+		is_player = yes
+		NOT = { is_country_type = decentralized }
+		scope:target_country = {
+			NOT = { is_country_type = decentralized }
+			is_player = yes
+		}
+		has_diplomatic_relevance = scope:target_country
+	}
+
+	possible = {
+		always = yes
+		custom_tooltip = {
+			text = DIPLO_ACTION_RELATIONS_COLD_OR_LESS_NOT
+			scope:target_country.relations:root > relations_threshold:cold
+		}	
+	}
+
+	accept_effect = {
+		change_relations = {
+			country = scope:target_country
+			value = {
+				add = relations_threshold:cold
+				subtract = scope:target_country.relations:root
+			}
+		}
+	}
```

Comment:
I'm having a hard time doing the math on the two accept effects...
You change the relations by `-30 - X`, where X is the current relations. But only if its more than -30..?

So if the relations is at -29, wouldnt this do nearly nothing?

I can see that the players must be hostile, but still...
Perhaps I'm misunderstanding something about basic math :s

Reply:
it goes to a fixed point (+/-50). It is meant to be used when relations are high and it will take a long time to change them. This is not a replacement for actually damaging relations or expelling diplomats.

And yes at -49 it would only give one less relation, on the flip side the other one would give 99 relations in the same scenario,

Reply:
Alright, so -50 - -49. Since minus minus gives plus, doesnt this mean that.. It.. Hmm... Alright, seems the math checks out, it would bring it to -50 no matter what :)